lib/xml-template-builder.js

```diff
@@ -1,236 +1,245 @@
 "use strict";
 
 const jstoxml = require("jstoxml");
-const _ = require("lodash");
 
-const xml = function() {
-  const DISPLAY_NAME = "S3rver";
-  const buildQueryContentXML = function(data, options) {
-    let content = _.map(data.objects, function(item) {
-      return {
-        Contents: {
-          Key: item.key,
-          LastModified: item.modifiedDate,
-          ETag: '"' + item.md5 + '"',
-          Size: item.size,
-          StorageClass: "Standard",
-          Owner: {
-            ID: 123,
-            DisplayName: DISPLAY_NAME
-          }
-        }
-      };
-    });
-    content = content.concat(
-      _.map(data.commonPrefixes, function(prefix) {
-        return {
-          CommonPrefixes: { Prefix: prefix }
-        };
-      })
-    );
-    content.unshift({
+const DISPLAY_NAME = "S3rver";
+
+function buildQueryContentXML(data, options) {
+  const content = [
```

Comment:
Could probably express this without needing to call `push`:

```javascript
const content = [
  { /* ... */ },
  ...data.objects.map(/* ... */),
  ...data.commonPrefixes.map(/* ... */)
]
```

Reply:
Since you bring that up, this whole function can be inlined since it's only used in `buildBucketQuery`.